com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -456,6 +459,26 @@ internal set
                 }
             }
 
+            /// <summary>
+            /// Returns whether this state update was an axial frame synchronization
+            /// (only applies when UseUnreliableDeltas is enabled)
+            /// </summary>            
+            public bool IsAxisFrameSync()
+            {
+                return UnreliableFrameSync;
+            }
+
+            /// <summary>
+            /// Returns whether this state update was sent with unreliable delivery.
+            /// If false, then it was sent with reliable delivery.
+            /// (only applies when UseUnreliableDeltas is enabled)
+            /// 
+            /// </summary>            
+            public bool IsUnreliableStateUpdate()
```

Comment:
This method also seems odd to me, because the name implies that it returns something about a particular update, but it actually returns a general statement on whether or not we are using unreliable transmission.

This part is a bit nitpicky, but I generally dislike having two boolean fields/properties/methods where one just returns the negation of the other. I think again I'd prefer to give the existing property a more public-API-appropriate name (like `UseReliableUpdates`) and make it public, rather than making something new, and let users who need that data handle the negation.